examples/01_tutorials/plot_single_subject_single_run.py

```diff
@@ -65,38 +65,13 @@
 # ------------------------------------
 #
 # We must now provide a description of the experiment, that is, define the
-# timing of the auditory stimulation and rest periods. According to
-# the documentation of the dataset, there were sixteen 42s-long blocks --- in
-# which 6 scans were acquired --- alternating between rest and
-# auditory stimulation, starting with rest.
-#
-# The following table provide all the relevant informations:
-#
-
-"""
-duration,  onset,  trial_type
-    42  ,    0  ,  rest
-    42  ,   42  ,  active
-    42  ,   84  ,  rest
-    42  ,  126  ,  active
-    42  ,  168  ,  rest
-    42  ,  210  ,  active
-    42  ,  252  ,  rest
-    42  ,  294  ,  active
-    42  ,  336  ,  rest
-    42  ,  378  ,  active
-    42  ,  420  ,  rest
-    42  ,  462  ,  active
-    42  ,  504  ,  rest
-    42  ,  546  ,  active
-    42  ,  588  ,  rest
-    42  ,  630  ,  active
-"""
-
-###############################################################################
-# We can read such a table from a spreadsheet file  created with OpenOffice Calcor Office Excel, and saved under the *comma separated values* format (``.csv``). 
+# timing of the auditory stimulation and rest periods. This is typically
+# provided in an events.tsv or events.csv file. The path of this file is
+# provided in the dataset.
+# Please note that events are sometimes called paradigms, though this
```

Comment:
I'd remove that statement. I don't think we should explicitly argue about terminology